framework/src/fvkernels/FVFluxKernel.C

```diff
@@ -45,9 +45,29 @@ FVFluxKernel::FVFluxKernel(const InputParameters & params)
 // constant monomial elements. We only have one quadrature point regardless of
 // problem dimension and just multiply by the face area.
 
+bool
+FVFluxKernel::skipForBoundary(const FaceInfo & fi)
+{
+  if (!fi.isBoundary())
+    return false;
+
+  std::vector<FVDirichletBC *> dirichlet_bcs;
+  auto q = _app.theWarehouse()
```

Comment:
Looks like you're making a copy of a vector here, and then not using it? I'm curious why we don't get any -Wunused errors

Reply:
Yeah - this is copy paste sloppiness. Fixed.